crossbar/router/broker.py

```diff
@@ -261,7 +261,7 @@ def on_authorize_success(authorized):
                             for receiver in receivers:
                                 if me_also or receiver != session:
                                     # the receiving subscriber session might have been lost in the meantime ..
-                                    if receiver._transport:
+                                    if receiver._session_id and receiver._transport:
                                         receiver._transport.send(msg)
```

Comment:
Would it be better here to use an existing interface? E.g. is it possible to say "all observers implement ITransport" or similar, and call ``.isOpen``.